Translations/pt_BR.po

```diff
@@ -477,7 +477,7 @@ msgid ""
 "Normal Difficulty\n"
 "This is where a starting character should begin the quest to defeat Diablo."
 msgstr ""
-"Dificuldade Normal\n"
+"Dificuldade normal\n"
```

Comment:
Why the lowercase? "Normal" may be an adjective on this case, but "Pesadelo" and "Inferno" are not. IMO they should be in uppercase.